tests/openshift.yml

```diff
@@ -0,0 +1,13 @@
+parameters:
+  vcluster:
+    base:
+      service_cidr: 172.30.0.0/16
+
```

Comment:
In case you intend to run this in GH actions then you'll need to adjust the config in .sync.yml and GH workflows